src/System.Management.Automation/engine/ComInterop/IDispatchMetaObject.cs

```diff
@@ -23,7 +23,7 @@ internal IDispatchMetaObject(Expression expression, IDispatchComObject self)
 
         public override DynamicMetaObject BindInvokeMember(InvokeMemberBinder binder, DynamicMetaObject[] args)
         {
-            Requires.NotNull(binder, nameof(binder));
+            ArgumentNullException.ThrowIfNull(binder);
```

Comment:
The file is copied from .Net Runtime repo. Please keep it unchanged.

Reply:
reverted